runtime/doc/syntax.txt

```diff
@@ -5353,6 +5353,8 @@ PmenuMatchSel	Popup menu: Matched text in selected item. Combined with
 		|hl-PmenuMatch| and |hl-PmenuSel|.
 							*hl-ComplMatchIns*
 ComplMatchIns	Matched text of the currently inserted completion.
+							*hl-PreInsert*
+PreInsert	Text inserted during autocompletion when "preinsert".
```

Comment:
Why didn't they use a `Compl` prefix, and spelled the suffix `Insert` whereas `ComplMatchIns` spells it as `Ins` :(